NetTopologySuite/Utilities/BitTweaks.cs

```diff
@@ -48,7 +42,7 @@ internal static float ReverseByteOrder(float value)
 
         internal static double ReverseByteOrder(double value)
         {
-            return DoubleBitsConverter.Int64BitsToDouble(ReverseByteOrder(DoubleBitsConverter.DoubleToInt64Bits(value)));
+            return System.BitConverter.Int64BitsToDouble(ReverseByteOrder(System.BitConverter.DoubleToInt64Bits(value)));

```

Comment:
Because `NetTopologySuite.Utilities.BitConverter` exists.  Obsoleting that one now.